example/README.md

```diff
@@ -1,9 +1,12 @@
 # Example Setup
 
 0. Prerequisites:
-    - Python 3
-    - Docker and `docker-compose`
+    - Python 3.8+
+    - docker compose V2
```

Comment:
This is not a hard requirement, maybe just mention that users might need to use `docker compose` instead of `docker-compose` depending on their setup?

Reply:
Understand the point, but docker-compose has been completely retired [one year ago](https://docs.docker.com/compose/migrate/), therefore no one should be encouraged to use it anymore.

Reply:
TIL! 💡 Sorry, I was out of the loop on that 😅